### ISSUE TITLE
Regions in X should maintain topic links

### ISSUE THREAD

**Issue by zang0** [socrata/opendatanetwork.com#334]
goto: 

http://opendatanetwork-staging.herokuapp.com/region/0400000US53/Washington/consumption/personal_consumption_expenditure/2014?

note:

All the Places in X links in the left column point to the default page, which in turn points to /population. We want to preserve the tab selection as best we can, so that Google has a strong signal that these other tabs matter. Currently, top internal links are all pointing to population demographics, which are tougher queries to win. We should be spreading this link love out.

![screen shot 2016-03-15 at 9 50 29 am](https://cloud.githubusercontent.com/assets/1051776/13786098/677197d4-ea93-11e5-9ec6-416cd131524a.png)

To be more specific, these links:

California
Oregon
Hawaii
Alaska

should point to:

/region/0400000US06/California/consumption/personal_consumption_expenditure/2014

not:

/region/0400000US06/California

_1 for states: perhaps we can start simple, and just include the deep topic links where we know they work, that is, into other states

Lets discuss this one though, as we don't want broken links. I suspect we might want a longer term strategy that points into` /$tab_name` and we rely on that page to forward to the default UNLESS we know the subcategory page exists, in which case we link directly.

**Comment by aaasen** [comment]
To solve this, I made it so selecting an invalid vector or a vector with no data will redirect to the default vector instead of triggering a 404. This means that it is safe to link to a specific vector, variable, and year without having to worry about whether or not the data exists.

All links in the left sidebar will maintain the selected vector, variable, and year. These links are automatically updated when a new variable or year is selected from the dropdown. I think it makes the experience a lot more coherent. Good change.

Closing. LMK if there are any issues.
